alicloud/table_alicloud_vpc_route_entry.go

```diff
@@ -0,0 +1,253 @@
+				Description: "The type of the next hop.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "ip_version",
+				Description: "The version of the IP protocol.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "destination_cidr_block",
+				Description: "The destination Classless Inter-Domain Routing (CIDR) block of the route entry.",
+				Type:        proto.ColumnType_CIDR,
+			},
+			{
+				Name:        "next_hop_region_id",
+				Description: "The region where the next hop instance is deployed.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "next_hop_oppsite_type",
+				Description: "The type of the next hop.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "next_hop_oppsite_region_id",
+				Description: "The region where the next hop instance is deployed.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "next_hops",
+				Description: "The information about the next hop.",
+				Type:        proto.ColumnType_JSON,
+				Transform:   transform.FromField("NextHops.NextHop"),
+			},
+
+			// Steampipe standard columns
+			{
+				Name:        "title",
+				Description: ColumnDescriptionTitle,
+				Type:        proto.ColumnType_STRING,
+				Hydrate:     getAwsVpcRouteEntryTurbotData,
+			},
+			{
+				Name:        "akas",
+				Description: ColumnDescriptionAkas,
+				Type:        proto.ColumnType_JSON,
+				Hydrate:     getAwsVpcRouteEntryTurbotData,
+			},
+
+			// Alicloud standard columns
+			{
+				Name:        "region",
+				Description: ColumnDescriptionRegion,
+				Type:        proto.ColumnType_STRING,
+				Hydrate:     getAwsVpcRouteEntryTurbotData,
+			},
+			{
+				Name:        "account_id",
+				Description: ColumnDescriptionAccount,
+				Type:        proto.ColumnType_STRING,
+				Hydrate:     getCommonColumns,
+				Transform:   transform.FromField("AccountID"),
+			},
+		},
+	}
+}
+
+//// LIST FUNCTION
+
+func listVpcRouteEntries(ctx context.Context, d *plugin.QueryData, h *plugin.HydrateData) (interface{}, error) {
+	region := plugin.GetMatrixItem(ctx)[matrixKeyRegion].(string)
+
+	// Create service connection
+	client, err := VpcService(ctx, d, region)
+	if err != nil {
+		plugin.Logger(ctx).Error("alicloud_vpc_route_entry.listVpcRouteEntries", "connection_error", err)
+		return nil, err
+	}
+
+	// Get VPC Route Table details
+	routeTable := h.Item.(routeTableRowData)
+
+	request := vpc.CreateDescribeRouteEntryListRequest()
+	request.Scheme = "https"
+	request.RouteTableId = routeTable.RouteTableId
+
+	response, err := client.DescribeRouteEntryList(request)
+	if err != nil {
+		plugin.Logger(ctx).Error("alicloud_vpc_route_entry.listVpcRouteEntries", "query_error", err, "request", request)
+		return nil, err
+	}
+	for _, i := range response.RouteEntrys.RouteEntry {
+		d.StreamLeafListItem(ctx, i)
+	}
+
+	return nil, nil
+}
+
+//// HYDRATE FUNCTIONS
+
+// Error: 2 list calls returned errors:
```

Comment:
Is there a purpose of leaving this commented out function in here?

Reply:
removed as discussed